### ISSUE TITLE
[Feature/Idea]: Prompt for update command

### ISSUE THREAD

**Issue by CoolPlayLin** [russellbanks/Komac#417]
### What would you like to see changed/added?

I want to use command line prompt instead of full command to use Komac sometimes. 

![image](https://github.com/russellbanks/Komac/assets/96709586/0ab09b2e-a815-4110-a702-69381266344a)

**Comment by hlqviet** [comment]
Yes, please. Installer architecture cannot be detected properly if I use the `--urls` parameter with multiple URLs.

**Comment by russellbanks** [comment]
> Yes, please. Installer architecture cannot be detected properly if I use the `--urls` parameter with multiple URLs.

@hlqviet Do you have an example?

**Comment by hlqviet** [comment]
> > Yes, please. Installer architecture cannot be detected properly if I use the `--urls` parameter with multiple URLs.
> 
> @hlqviet Do you have an example?

Here it is: microsoft/winget-pkgs#137181
I had to manually edit the URL.

**Comment by russellbanks** [comment]
The issue with the arm64 one is that it says arm64ec. Komac has this process of finding architectures in URLs:

1. check for {delimiter}{architecture}{delimiter}
2. check for {architecture}.{extension}

The architectures are in a pre-set list, so I just need to add arm64ec to it. Searching for the architecture without any other patterns doesn't work great because the app `ArmCord.ArmCord` identifies as an arm architecture when it's not.

**Comment by drshtmstry** [comment]
Previous versions had command line prompts for `komac update` which was better for new users and creating manifests without submitting.
It provided step-by-step guide for each parameters & for each installer.

**Comment by G0rocks** [comment]
I like the idea of having step by step prompts for komac update but at least I suggest changing from:
```
komac update --help
Add a version to a pre-existing package

Usage: komac update [OPTIONS] --version <PACKAGE_VERSION> --urls <URLS>... <PACKAGE_IDENTIFIER>
```

to:
```
komac update --help
Add a version to a pre-existing package

Usage: komac update <PACKAGE_IDENTIFIER> [OPTIONS] --version <PACKAGE_VERSION> --urls <URLS>
```

Because when I ran `komac update --version <PACKAGE_VERSION> --urls <URLS>... <PACKAGE_IDENTIFIER>` resulted in:
error: invalid value '...' for '--urls <URLS>...': relative URL without a base

For more information, try '--help'.

And it turns out you have to put the package identifier immediately after "komac update" in order for the command to work.
